public/index.css

```diff
@@ -25,6 +25,7 @@
     border: black 1px solid;
     border-radius: 5%;
     background-image: url('./image/dish-image.jpeg'); 
+    background-image: linear-gradient(rgba(0, 0, 0, 0.3), rgba(0, 0, 0, 0.3)), url('./image/dish-image.jpeg'); 
```

Comment:
Added a slight gradient that we can increase of decrease to the background image for accessibility and aesthetic purposes. @josueJURE